tests/scripts/git_skip_ci_globs.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import argparse
+import fnmatch
+from typing import Optional
+
+from git_utils import git
+
+
+globs = [
+    "*.md",
+    "docker/**",
+    "conda/**",
+    ".github/**",
+    ".asf.yaml",
+    "LICENSE",
+    "NOTICE",
+    "KEYS",
+]
+
+
+def match_any(f: str) -> Optional[str]:
+    for glob in globs:
+        if fnmatch.fnmatch(f, glob):
```

Comment:
i think this doesn't quite match patterns like `glob.glob` does--specifically, `**` doesn't work. i'm not sure it entirely matters here, but if we do want that flexibility, we may want to find a better way to do this

Reply:
ah good catch, fnmatch doesn't need `**` to search recursively so I fixed the patterns

Reply:
want to add a note to the constant above? can be follow-on

Reply:
planning to address in https://github.com/apache/tvm/issues/10472